server/modules/instantiation/jobs.js

```diff
@@ -417,7 +424,7 @@ module.exports = function (config, portProvider, interpolationHelper, buildInsta
                 buildInstance.log('Running docker-compose.');
 
                 var dockerCompose = spawn(
-                    'docker-compose', ['--file', dockerComposeFileName, 'up', '--abort-on-container-exit', '--no-color'],
+                    'docker-compose', ['--file', dockerComposeFileName, 'up', '-d', '--no-color'],
```

Comment:
I think it's fine as it is. Previously running `docker-compose` was the last stage so I didn't need to detach this.